devices/inovelli.js

```diff
@@ -312,6 +321,34 @@ const ATTRIBUTES = {
         description:
       'For use with Smart Bulbs that need constant power and are controlled via commands rather than power.',
     },
+    doubleTapUptoParam55: {
+        ID: 53,
+        dataType: BOOLEAN,
+        displayType: 'enum',
+        values: {'Disabled': 0, 'Enabled': 1},
+        description: 'Enable or Disable setting brightness to parameter 55 on double-tap UP.',
+    },
+    doubleTapUptoParam56: {
+        ID: 54,
+        dataType: BOOLEAN,
+        displayType: 'enum',
+        values: {'Disabled': 0, 'Enabled': 1},
+        description: 'Enable or Disable setting brightness to parameter 56 on double-tap DOWN.',
+    },
+    brightnessLevelForDoubleTapUp: {
+        ID: 55,
+        dataType: UINT8,
+        min: 0,
+        max: 100,
```

Comment:
shouldn't this be 255/254?

Reply:
@EricM_Inovelli says params 55 and 56 are both 0-100. See his post here:
https://community.inovelli.com/t/blue-series-2-1-firmware-changelog-vzm31-sn/12326/372

Reply:
ok - in that case at least align line 349 to match this one

Reply:
> Apparently the 0-100 ranges @EricM_Inovelli provided for parameters 55 & 56 were a result of him lifting the parameter definitions directly from the Hubitat driver, which uses percentages rather than a 2-byte number converted to decimal. Since the z2m converter does not convert to percentages, the 0-100 range for parameters 55 and 56 is not correct. Please adjust the max value for parameters 55 and 56 to be 254. Forum user mamber tested all the way to 255, and found that the switch only responds if the values for these parameters is in the range 0-254. See this forum post: https://community.inovelli.com/t/blue-series-2-1-firmware-changelog-vzm31-sn/12326/425

Yep, sorry, I got those parameter definitions from the Hubitat driver. The actual permitted values are:

![image](https://user-images.githubusercontent.com/37669481/231577267-14e8dd7d-e1d2-4f1c-bb73-0a018531b387.png)